go.sum

```diff
@@ -1,6 +1,8 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/Shopify/ejson v1.3.1 h1:Oawj8bu0IhjvL2W4/wzHJiGRswWTZkOU4RmjJW+JQCg=
```

Comment:
Nit: since we upgrade to `v1.3.2`, I'd expect this to be missing.

If you run `go mod tidy` is it removed? 🧹